blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Sourcegraph 3.6: Bitbucket Server plugin, search performance improvements, and quick links'
+author: Christina Forney
+publishDate: 2019-07-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.6
+heroImage: /sourcegraph-mark.png
+published: true
+---
+
+[Sourcegraph](https://about.sourcegraph.com/) is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+<div style="padding-left: 2rem">
+
+[**🌎 Sourcegraph plugin provides native code intelligence to Bitbucket Server**](#)<br />
+
+[**📉 30% indexed search latency improvement for small and medium instances**](#)<br />
+
+[**🔗 Add quick links to repositories or important resources below the search bar**](#)<br />
```

Comment:
```suggestion
[**🔗 New customizable quick links for frequently used code searches or other URLs**](#)<br />
```

Suggested this change as I imagine most would use it for searches they use frequently, and I don't think you need to mention in this heading where the quick links go.